common/lib/src/realm_common_base.dart

```diff
@@ -1,4 +1,4 @@
-////////////////////////////////////////////////////////////////////////////////
+// //////////////////////////////////////////////////////////////////////////////
```

Comment:
I assume this is to make pease with dart doc. Ie. it doesn't start with /// .. but it does look like an error. I suggest we use another "frame" eventually